cloudformation/iam_federation.js

```diff
@@ -92,15 +105,16 @@ function removeStatement(input, callback) {
   console.log(assumeDoc.Statement);
   console.log(input.roleArn);
   var found = -1;
-  for (var i = 0; i < assumeDoc.Statement.length; i++) {
```

Comment:
I have addressed it in the addStatement and so statement we searching for will be in an array and not a string.